qsr_lib/src/qsrlib_qsrs/qsr_qtc_bc_simplified.py

```diff
@@ -134,6 +146,7 @@ def make(self, *args, **kwargs):
                         qsr=self.qtc_to_output_format((qtc), kwargs["future"])
                     )
                     ret.add_qsr(qsr, idx+1)
+                combinations[between] = qtc_sequence
```

Comment:
This can be avoided with the rest of the if by using a try-except clause and the mapping of reversed relations which I am +1 if needed to speed up performance (although gives an extra layer of development mistake if they are wrongly inputted). E.g. in mixed python/pseudocode
```python
try:
  qsr = inverse_mapping[ret.qsrs[inversed_between]["qtcbs"]]
except KeyError:
  qsr = normal computation
```

Safer when `--future` is standard.